src/StaticCaching/StaticCacheManager.php

```diff
@@ -67,13 +67,21 @@ public function flush()
     {
         $this->driver()->flush();
 
-        collect($this->cacheStore()->get('nocache::urls', []))->each(function ($url) {
-            $session = $this->cacheStore()->get($sessionKey = 'nocache::session.'.md5($url));
-            collect($session['regions'] ?? [])->each(fn ($region) => $this->cacheStore()->forget('nocache::region.'.$region));
-            $this->cacheStore()->forget($sessionKey);
-        });
+        try {
```

Comment:
I wanted to try and check the name of the store returned by `$this->cacheStore()` but I couldn't find a way to get the cache store's name so reverted to this `try {} catch {}` solution.